### DISCUSSION TITLE
Allow multiple stacks in BarChart

### DISCUSSION THREAD

**Discussion by tikoflano** [mantinedev/mantine#5802] [Feature requests]
When using a [stacked BarChart](https://mantine.dev/charts/bar-chart/#stacked-bar-chart) you might want to show multiple stacks. Currently all data is grouped in the same stack so it is not possible.

I think the solution would be:
1. Add a new optional key to [`BarChartSeries`](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/charts/src/BarChart/BarChart.tsx#L37):

```js
export interface BarChartSeries extends ChartSeries {
  stack?: string;
}
```
2. [Use](https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/charts/src/BarChart/BarChart.tsx#L185) the new `stack` key if defined, or fallback to `"stack"`

```js
...
stackId={stacked ? (item.stack ?? 'stack') : undefined}
...
```

An example use case, when you have purchases and sales values by type and want to show them side by side, like:

```js
const series = [
  {name: "games", amount: 20, color: "green.2", stack: "sales"},
  {name: "games", amount: 10, color: "green.2", stack: "purchases"},
  {name: "food", amount: 2, color: "red.2", stack: "sales"},
  {name: "food", amount: 0, color: "red.2", stack: "purchases"},
]
```

**Comment by rtivital** [comment]
This feature will be available in 7.6 with `barProps` prop.
